### ISSUE TITLE
Feature Request: Add metadata "Keywords" as input 

### ISSUE THREAD

**Issue by avviano** [thedyze/save-image-extended-comfyui#19]
There is currently no ComfyUI node for SAVING IMAGE available anywhere that lets you save keywords generated by WD14 Tagger to the internal PNG metadata. Would be great if this could be added as WIDGET/INPUT.

This is the missing link to be able to use ComfyUI to automatically tag images with keywords. 

Its possible to auto-generate keyword tags for any LOAD IMAGE, using either "MS Kosmos-2 Interrogator" or "WD14 Tagger" or "CXH_JoyTag", but its not possible in any available node to actually save these keywords into the internal PNG KEYWORD metadata. 

Thanks.

**Comment by audioscavenger** [comment]
i agree and it seems that it is possible indeed, according to [PNG specs](https://exiftool.org/TagNames/PNG.html). please post your request [here](https://github.com/audioscavenger/save-image-extended-comfyui/discussions) - but i must warn you, this does not look easy. I do it easy for Jpeg, with exiftool command line. but from within Comfy + python, that's another story.
